.travis.yml

```diff
@@ -155,6 +155,15 @@ jobs:
                             --disable-test-cpp --enable-test-vald --enable-test-helper --enable-test-perf --enable-test-perf-proc --enable-test-example
                           - make -j $(nproc)
                 - stage: test
+                  compiler: "\"clang-3.8 --target=aarch64-linux-gnu\""
+                  env: TEST="clang-3.8 aarch64-linux-gnu" CROSS_ARCH="arm64"
```

Comment:
@Bill-Fischofer-Linaro I do not know. I used clang-3.8, as the rest of Travis configuration defaults to this version.